lib/features/rules/BpmnRules.js

```diff
@@ -128,6 +128,10 @@ BpmnRules.prototype.init = function() {
         return canConnect(element.source, element.target, element);
       }
 
+      if (element.host) {
+        return !!canAttach(element, element.host, null, position);
```

Comment:
Can we drop the `!!` here?

Reply:
Yes, as soon as we merge https://github.com/bpmn-io/min-dash/pull/14 and update `min-dash`. Otherwise, we need to somehow handle the case that the rule might return `"attach"` or `true` based only on the selection order.